Core/include/Acts/Geometry/ProtoDetector.hpp

```diff
@@ -0,0 +1,106 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Extent.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Utilities/BinningData.hpp"
+
+#include <string>
+
+namespace Acts {
+
+/// A proto volume description being used to define an overall
+/// structure of either a TrackingVolume or Experimental::DetectorVolume
+struct ProtoVolume {
+  /// Name of the proto volume
+  std::string name = "";
+  /// The extent of this layer
```

Comment:
```suggestion
  /// The extent of this volume
```
I got confused here. does the volume also have a single surface?

Reply:
Fixed.